Application/Frontend/pages/Member_List.tsx

```diff
@@ -19,9 +20,11 @@ import { MemberList } from '@/components/Server/Member_List'; // Assuming the co
  *   accessible and user-friendly across various devices and user demographics.
  */
 export default function MemberListPage() {
+  const [chatID, setChatID] = useState<string>("b0b407caf9a1caaec74ed3f089ccb0916418e64984c9045b27361c920bff83df")
```

Comment:
I see you have specified a default chat as part of your testing. This is ok for now, and is something I will need to change to reflect the text channels that appear on the left sidebar when I integrate them. I will run a query to check if the signed in user is part of any chats. We can only render the member list if the user is currently looking at a chat (will be a bit difficult to do), in which case the default will just be an empty string or the chat we are currently looking at.